src/mlpack/tests/cv_test.cpp

```diff
@@ -187,10 +187,31 @@ TEST_CASE("R2ScoreTest", "[CVTest]")
 
   double expectedR2 = 0.99999779;
 
-  REQUIRE(R2Score::Evaluate(lr, data, responses)
+  REQUIRE(R2Score<false>::Evaluate(lr, data, responses)
           == Approx(expectedR2).epsilon(1e-7));
 }
 
+/**
+ * Test the Adjusted R squared metric
```

Comment:
```suggestion
 * Test the Adjusted R squared metric.
```

Add stop to be consistent with the rest of the codebase.